docs/testing/uninstall-add-in.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Uninstall add-ins under development
+description: Learn how to prevent and uninstall ghost or orphaned add-ins you are developing.
+ms.topic: troubleshooting-problem-resolution
+ms.date: 12/28/2024
+ms.localizationpriority: medium
+---
+
+# Uninstall add-ins under development
+
+Incompletely removed add-ins under development can leave artifacts, such as custom ribbon buttons or registry entries, on your development computer. In the case of Outlook add-ins, these artifacts can be added to other computers when you sign into Outlook on them with the same ID as you used to develop the add-in. In this article, we call these ghost add-ins. 
+
+This article provides some guidance to minimize the chance of these problems and for resolving them if they occur.
+
+## Prevent the problems
+
+When an add-in is sideloaded, several things happen:
+
+- A web server, usually on localhost, is started to serve the add-ins files (HTML, CSS, JavaScript, etc.).
+- These same files are cached on your development computer.
+- The add-in is registered with the development computer. The registration is done with Registry entries on a Windows computer or with certain files saved to the file system on a Mac.
+- Most tools for sideloading add-ins also automatically open the Office application that the add-in targets and populate the application with any custom ribbon buttons or context menu items that are defined in the add-in's manifest.
+- For an Outlook add-in, the add-in's manifest and certain other values about the add-in are registered with the Exchange service.
+
+   > [!IMPORTANT]
```

Comment:
This whole section contains valuable information, but, for me, doesn't mesh with the header of "Prevent the problems". Is there a more logical place for this Outlook note?

Reply:
> This whole section contains valuable information, but, for me, doesn't mesh with the header of "Prevent the problems". Is there a more logical place for this Outlook note?

Moved it up to the introduction section.